haskell/test/Lorentz/Contracts/Test/FA2.hs

```diff
@@ -471,78 +471,69 @@ fa2Spec fa2Originate = do
 
   -- These tests require permission descriptor to be configured that allows for operator transfer.
   -- We have such a configuration set by default in defaultOriginationParams.
+  let
+    checkForOperator stablecoinContract owner operator expectation =
+      lExpectStorage stablecoinContract $ \storage -> do
+        let found = isJust $ M.lookup (owner, operator) (unBigMap $ sOperators storage)
+        if found /= expectation
+            then Left $ CustomTestError $ "Unexpected operator status. Expected: " <> (show expectation) <> " Found: " <> (show found)
```

Comment:
```suggestion
            then Left $ CustomTestError $ "Unexpected operator status. Expected: " <> show expectation <> " Found: " <> show found
```

Reply:
Also please split it into two lines, this one looks too long.

Reply:
> Also please split it into two lines, this one looks too long.

Have changed.